CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Enhancements
 
+- Added `CheckNullifiersByPrefix` endpoint (#419).
```

Comment:
nit: we usually sort these in descending order (so new PRs get added at the end of the list).